main.py

```diff
@@ -1,8 +1,8 @@
-from components import Network, Link, Host
+from components import Network, Link, Host, Router
 
 if __name__ == '__main__':
     H1 = Host("H1")
     H2 = Host("H2")
-    L1 = Link("L1", 10.0,10,64, H1, H2)
+    L1 = Link("L1", 10.0, 10, 64, H1, H2)
     network = Network([H1, H2], [], [L1], [])
-    network.run()
+    # network.run()
```

Comment:
I commented it out when I was debugging routing since I wasn't using the test case. I'll revise.